### ISSUE TITLE
 关于太乙模型加载的问题

### ISSUE THREAD

**Issue by dading105** [IDEA-CCNL/Fengshenbang-LM#331]
使用半精度 Half precision FP16 (CUDA)

import torch
from diffusers import StableDiffusionPipeline
torch.backends.cudnn.benchmark = True
pipe = StableDiffusionPipeline.from_pretrained("IDEA-CCNL/Taiyi-Stable-Diffusion-1B-Chinese-v0.1", torch_dtype=torch.float16)
pipe.to('cuda')

prompt = '飞流直下三千尺，油画'
image = pipe(prompt, guidance_scale=7.5).images[0]
image.save("飞流.png")

生成图片 报错
text_config_dict is provided which will be used to initialize CLIPTextConfig. The value text_config["id2label"] will be overriden.
/home/dt/.conda/envs/pytorch/lib/python3.11/site-packages/transformers/models/clip/feature_extraction_clip.py:28: FutureWarning: The class CLIPFeatureExtractor is deprecated and will be removed in version 5 of Transformers. Please use CLIPImageProcessor instead.
warnings.warn(
100%|████████████████████████████████████████| 50/50 [02:23<00:00, 2.88s/it]
/home/dt/.conda/envs/pytorch/lib/python3.11/site-packages/diffusers/utils/pil_utils.py:38: RuntimeWarning: invalid value encountered in cast
images = (images * 255).round().astype("uint8")
Potential NSFW content was detected in one or more images. A black image will be returned instead. Try again with a different prompt and/or seed.

生成黑色突破，请问如何解决？

**Comment by dading105** [comment]
黑色图片

**Comment by ganzhiruyi** [comment]
黑色图片是触发了NSFW，可以参考transformer里尝试关掉。https://huggingface.co/docs/diffusers/main/en/using-diffusers/loading#safety-checker
`
from diffusers import DiffusionPipeline

repo_id = "runwayml/stable-diffusion-v1-5"
stable_diffusion = DiffusionPipeline.from_pretrained(repo_id, safety_checker=None)
`

**Comment by dading105** [comment]
> 黑色图片是触发了NSFW，可以参考transformer里尝试关掉。https://huggingface.co/docs/diffusers/main/en/using-diffusers/loading#safety-checker
> 
> ```
> from diffusers import DiffusionPipeline
> 
> repo_id = "runwayml/stable-diffusion-v1-5"
> stable_diffusion = DiffusionPipeline.from_pretrained(repo_id, safety_checker=None)
> ```

关了也不行
